docs/DeveloperGuide.md

```diff
@@ -275,6 +275,106 @@ as values.<br>
   * Cons: Updating `UniqueTagList` takes O(logn) time every time; requires additional data structure to maintain
   `UniqueTagList` accurately.
 
+### Recently Contacted Information feature
```

Comment:
Can you put below Date features

Reply:
Also, change name to Interaction record features

Reply:
I think Interaction record is what we market. Like we, developers don't call name `Person` class as `Contact`

Reply:
Okay